### ISSUE TITLE
Monitor benchmark suggestion

### ISSUE THREAD

**Issue by danielballan** [NSLS-II/caproto#310]
From Michael Davidsaver (recalled from conversation by me, I hope accurately):

> Serve one PV that updates at a slow, steady rate. Serve a second PV whose update rate ramps slowly. Observe at what point the client has to start dropping messages from the ramping PV and, further, verify that it is processing the update from the steady PV at an even rate. This checks that the client is "fair."

**Comment by hhslepicka** [comment]
That is right! I can confirm this talk and the issue reflects the discussed problem. ;)
